.github/workflows/build.yaml

```
@@ -80,7 +80,8 @@ jobs:
       - shell: bash
         run: |
           [ "false" == ${{ matrix.quic_support }} ] && export BUILD_WITHOUT_QUIC=1
-          git config --global --add safe.directory /__w/emqtt-bench/emqtt-bench
+          git config --global --add safe.directory $(pwd)
```

`quicer` sometimes also complains about safe dirs:

```suggestion
          git config --global --add safe.directory "*"
```